util/chunk/column.go

```diff
@@ -605,3 +606,16 @@ func (c *Column) CopyReconstruct(sel []int, dst *Column) *Column {
 	}
 	return dst
 }
+
+// WriteTo writes the raw data in the specific row to w.
+func (c *Column) WriteTo(rowID int, w io.Writer) (int, error) {
+	var data []byte
+	if c.isFixed() {
+		elemLen := len(c.elemBuf)
+		data = c.data[rowID*elemLen : rowID*elemLen+elemLen]
+	} else {
+		start, end := c.offsets[rowID], c.offsets[rowID]
```

Comment:
Does it seem to be wrong that `start == end`, any test case covered this?

Reply:
The benchmark was fake because of this bug. 😢
I'll update a new one.